test/clj/rems/browser_test_util.clj

```diff
@@ -195,23 +195,26 @@
         form (test-data/create-all-field-types-example-form! "owner" {:organization/id "nbn"} "Example form with all field types" {:en "Example form with all field types"
                                                                                                                                    :fi "Esimerkkilomake kaikin kenttätyypein"
                                                                                                                                    :sv "Exempelblankett med alla fälttyper"})
-        _simple-form (test-helpers/create-form! {:actor "owner"
+        private-field-form (test-helpers/create-form! {:actor "owner"
```

Comment:
context on this change: as i discovered during this PR we originally used single id for application form fields called `application-fields`, but if application contained multiple forms via multiple catalogue items or workflow, this id would be repeated multiple times, causing accessibility violation.

question remains, is this data needed in the enclosing function? i guess this function is used to populate some data, perhaps local dev db?

Reply:
Do you mean do we need ~~_simple-form~~ `private-field-form`? The `create-test-data` also creates whatever test data we have in a running dev/test instance so sometimes we add stuff to be able to manually demo technical things (separate from actual demo data that could be more user oriented and clean).

Reply:
yeah, that's what i was thinking. i guess this could demo multiple forms in one application? although "private field form" is not very useful field but it should clarify the privacy point when viewing the application as reviewer

Reply:
Yeah it'd be very useful for the future to have clear test data with multiple forms (through catalogue items / workflow).

Reply:
this is now in